nbgrader/exchange/exchange.py

```diff
@@ -42,6 +42,17 @@ class Exchange(LoggingConfigurable):
         help="Format string for timestamps"
     ).tag(config=True)
 
+    @validate('timestamp_format')
+    def _valid_timestamp_format(self, proposal):
+        try:
+            from dateutil.parse import parse
```

Comment:
Can these imports be moved to the top of the file?

Reply:
Yes, this was only to put things together in this PR